<?xml version="1.0"?>
<launch>
    
    <node name="usb_cam_reader" pkg="usb_cam_reader" type="usb_cam_reader_node" output="screen">
        <param name="cam_id" value="1" />
        <param name="camera_name" value="logitech_c310_1280_960_laser" />
    </node>
    
    <group ns="usb_cam_reader">
        <node pkg="image_proc" type="image_proc" name="image_proc" />
    </group>
<!--    
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam_reader/image_rect_color"/>
    <param name="autosize" value="true" />
    <param name="image_transport" value="compressed"/> 
  </node>-->
  
<!--    
    <node name="usb_cam_reader2" pkg="usb_cam_reader" type="usb_cam_reader_node" output="screen">
        <param name="cam_id" value="2" />
        <param name="camera_name" value="logitech_c310_1280_960" />
    </node>-->
    
<!--    <group ns="laser_1">
        <node name="laser_1" pkg="lms1xx" type="lms1xx">
            <param name="host" value="10.0.0.10"/>
            <param name="frame_id" value="/laser_1"/>
        </node>
    </group>-->

<!--     This is a launch file for recording the topics for my thesis  -->
    
    <node pkg="rosbag" type="record" name="rosbag_record"
    args="record -o /home/luis/Documentos/Video/Camera/line

    /usb_cam_reader/image_rect_color/compressed"/>
<!--     <node name="rqt_graph" pkg="rqt_graph" type="rqt_graph" output="screen"></node> -->
  
</launch>